src/screen/Drawer/DrawerScreen.tsx

```diff
@@ -29,6 +30,12 @@ function DrawerScreen({navigation}: DrawerContentComponentProps) {
           </TouchableOpacity>
         </View>
         <Layout style={styles.rest}>
+          <ListItem
+            title="Accounts"
+            accessoryLeft={(props) => <Icon {...props} name="person-outline" animation="zoom" />}
+            onPress={() => navigation.navigate(accountsNavigator)}
+          />
+          <Divider />
```

Comment:
@ahoseinian I am not sure how this change got in here.
Can you please confirm if it is ok?

Reply:
Since the dashboard is our main entry it makes sense to be first in the Drawer screen. You can move `Accounts` back

Reply:
hmm probably some problem in merge, but maybe you can move it under dashboard. I thought dashboard being the top must item is better.

Reply:
`Accounts` menu item is after dashboard now.
https://github.com/litentry/litentry-app/pull/443/commits/53671380409ec16635731e459ca0337273734f46